code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -506,6 +506,7 @@
 			if(allowed(usr))
 				locked = !locked
 				to_chat(user, "<span class='notice'>You [ locked ? "lock" : "unlock"] [src]'s cover.</span>")
+				to_chat(src, "<span class='notice'>[usr] [locked ? "locks" : "unlocks"] your cover.</span>")
```

Comment:
```suggestion
				to_chat(src, "<span class='notice'>[usr] [locked ? "locked" : "unlocked"] your cover.</span>")
```

Reply:
The unlocker message is in the present tense.  Why would the borg message be in the past tense?

Reply:
Good point